grails-app/services/de/iteratec/osm/measurement/environment/BrowserService.groovy

```diff
@@ -27,33 +28,37 @@ class BrowserService {
         return Collections.unmodifiableSet(result)
     }
 
-    List<Browser> findAllByNameOrAlias(List<String> browserNameOrAlias) {
+    List<Browser> findAllByName(List<String> browserNames) {
```

Comment:
The name is misleading as it suggests that you don't try to find it by Alias anymore.